website/docs/docs/deploy/ci-jobs.md

```diff
@@ -29,6 +40,37 @@ dbt Labs recommends that you create your Slim CI job in a dedicated dbt Cloud [d
 
 3. In the **Triggers** section, choose the **Continuous Integration** (CI) tab. Then, enable the **Run on Pull Requests** option. This configures pull requests and new commits to be a trigger for the Slim CI job.
 
+</TabItem>
+
+<TabItem value="beta" label="Beta version">
+
+To make CI job creation easier, many options on the **CI job** page are set to default values that dbt Labs recommends that you use. You can change some of these defaults and there are some that you can’t (which are greyed out).
+
+1. On your **Staging** environment page, click **Create Job** > **Continuous Integration Job** to create a new CI job. 
+
+2. Options in the **Job Description** section:
+    - **Job Name** &mdash; Specify the name for this CI job.
+    - **Environment** &mdash; By default, it’s set to the environment you created the CI job from.
+    - **Triggered by pull requests** &mdash; By default, it’s enabled.
+
+3. Options in the **Execution Settings** section:
+    - **Compare changes against an environment (Deferral)** &mdash; By default, it’s set to the **Production** environment if you created one. This option allows dbt Cloud to check the state of the code in the PR against the code running in the deferred environment, so as to only check the modified code, instead of building the full table or the entire DAG.
+    - **Commands** &mdash; By default, it includes the `dbt build --select state:modified+` command. This informs dbt Cloud to build only new or changed models and their downstream dependents. Importantly, state comparison can only happen when there is a deferred job selected to compare state to. Click **Add command** to add more [commands](/docs/deploy/job-commands)  that you want to be invoked when this job runs.
+    - **Generate docs on run** &mdash; Enable this option if you want to [generate project docs](/docs/collaborate/build-and-view-your-docs) when this job runs. This option is disabled by default since doc generation testing isn't typically performed for every CI check.
+    - **Add API trigger** &mdash; Set up an [API](/docs/dbt-cloud-apis/overview) trigger to run this CI job. In the POST request method, you must also include the pull request ID and `get_sha` value.
```

Comment:
waiting to hear back on exact field names here for PR ID and get_sha

Reply:
@knartovich will you share w/ Ly what the exact names of the fields are that are needed when you trigger a CI job via API? We need PR_ID and git_sha?

@nghi-ly looks like a typo w/ get_sha. But let's wait for Katia to share

Reply:
you can trigger your CI run without either of those fields, but if you want your run to post a status back to your PR/MR you need both. they are called `github_pull_request_id` & `git_sha`. looks like this functionality has not been implemented for GL or ADO yet

Reply:
okay great

"In the POST request method, you must also include the` github_pull_request_id` and `git_sha` if you would like to post the status of the check back to the git provider and have set up native integration set up through dbt Cloud"

Reply:
@knartovich I thought the post back requires a native integration w/ dbt Cloud and the git provider set up. I thought the PR ID and git sha were required so that we can build in temp schemas + auto-cancel stale runs, which we handle w/ our scheduler

Reply:
you are correct -- a native integration is required. but we only post back if the run trigger is associated with a pull request, & git SHA is required

PR ID is need for auto-cancellation; git SHA is not

as far as i can tell, temporary schema naming/dropping is done through webhooks. the user can pass `schema_override` if they wish, but we don't automatically create a temp schema until the CI run is triggered by a webhook

Reply:
with the latest UI, looks like this option is removed. updated wording to align